Sources/Benchmark/BenchmarkCommand.swift

```diff
@@ -54,7 +69,16 @@ internal struct BenchmarkCommand: ParsableCommand {
             throw ValidationError(debugBuildErrorMessage)
         }
         if iterations != nil && iterations! <= 0 {
-            throw ValidationError(iterationsErrorMessage)
+            throw ValidationError(positiveNumberError(flag: "--iterations", of: "integer"))
+        }
+        if warmupIterations != nil && warmupIterations! <= 0 {
+            throw ValidationError(positiveNumberError(flag: "--warmup-iterations", of: "integer"))
+        }
+        if maxIterations != nil && maxIterations! <= 0 {
+            throw ValidationError(positiveNumberError(flag: "--max-iterations", of: "integer"))
+        }
+        if minTime != nil && minTime! <= 0 {
+            throw ValidationError(positiveNumberError(flag: "--min-time", of: "floating"))
```

Comment:
```suggestion
            throw ValidationError(positiveNumberError(flag: "--min-time", of: "floating point number"))
```

(In concert with change below.)